app/renderer/src/containers/query-field-container/query-field-container.js

```diff
@@ -6,54 +6,47 @@ import { ThemeSchema } from '../../schema';
 import { IPC_WINDOW_SHOW, IPC_WINDOW_HIDE } from '../../../../ipc';
 
 const QueryFieldContainer = class extends Component {
+  queryField = null;
+
   static displayName = 'QueryFieldContainer';
-  constructor(props) {
-    super(props);
-    this.handleChange = this.handleChange.bind(this);
-  }
 
   componentDidMount() {
-    const self = this;
-    // focus the query input field when the window is shown
     ipcRenderer.on(IPC_WINDOW_SHOW, () => {
-      self.focus();
+      this.focus();
     });
-    // resets the querty and blurs the input field when the window is hidden
     ipcRenderer.on(IPC_WINDOW_HIDE, () => {
-      this.props.onReset && this.props.onReset();
-      self.blur();
+      if (this.props.onReset) this.props.onReset();
+      this.blur();
     });
   }
 
-  focus() {
```

Comment:
we should start preferring property initializers from here on

Reply:
+1